lib/src/ast.dart

```diff
@@ -434,3 +434,11 @@ class _ElementVisitorAdapter extends GeneralizingElementVisitor {
     }
   }
 }
+
+extension ElementExtension on Element? {
+  bool get isDartCorePrint {
+    var element = this;
+    if (element is! FunctionElement) return false;
```

Comment:
Even less important, but just note that `element is FunctionElement` would fit well into the following `&&` expression. Although there will be multiple lines :-)

Reply:
I do actually like that better.  Updated.  Thanks!